### DISCUSSION TITLE
Revert axes

### DISCUSSION THREAD

**Discussion by mtt09mtt** [biolab/orange3#5776] [General]
I am new to Orange.
Is there a way of making revert axes in scatter plot widget?
Thanks and Regards,
Thang,

**Comment by ajdapretnar** [comment]
Do you mean instead of going from, i.e. 0 to 100 to go from 100 to 0?
No, currently there's not. I also think such an option might lead to serious difficulties in reading the plots.

**Comment by markotoplak** [comment]
There are no plot options like this, but if you really want the visual effect, perhaps use Feature Constructor where you construct feature*(-1). 

May I ask what are you plotting? I have only seen one domain where a reversed axis is standard: spectroscopy. So I understand it might also be useful somewhere else.

**Comment by mtt09mtt** [comment]
Thank you all for the answers. I spent some time on it and found that there is no option of invert axis of scatter plot (Fig. 1) but with plot of polynomial regression it is! if right click on the axis (Fig 2).

![image](https://user-images.githubusercontent.com/75878093/149164132-1d47efc1-60cc-44a3-9dbe-22d74f9daa72.png)
![image](https://user-images.githubusercontent.com/75878093/149164301-00735b87-0698-485e-bafd-c3bf024d4985.png)

Regards,
